typespec-tests/tspconfig.yaml

```diff
@@ -11,4 +11,4 @@ options:
     dev-options:
       generate-code-model: true
       support-versioning: true
-      loglevel: info
+      loglevel: info
```

Comment:
What is the difference of this change?

Reply:
No difference here. I added and removed `debug: true` on my mac, probably cause this EOL difference. I'll see what I can do..

Reply:
removed